### ISSUE TITLE
Hello, I have an question about Label Smoothness Regularization

### ISSUE THREAD

**Issue by johnnyjana730** [hwwang55/KGCN#3]
I fix model parameter and train, eval, test set for every datasets to compare the score of one with and without Label Smoothness Regularization. However, the evaluation score of one with Label Smoothness Regularization don't outperform steadily the one without Label Smoothness Regularization. I want to ask that if it has some trick to use Label Smoothness Regularization rightly.

**Comment by hwwang55** [comment]
Hi there, what results did you get? Try running 5 or 10 times then compare the average performance.

**Comment by johnnyjana730** [comment]
hey that is my result

**Comment by johnnyjana730** [comment]
![image](https://user-images.githubusercontent.com/20666568/59237528-41fc9500-8c2d-11e9-9b4e-32aa20e5d03d.png)

**Comment by johnnyjana730** [comment]
Then, I use book dataset from your previous paper https://github.com/hwwang55/RippleNet/tree/master/data/book. That book dataset contains 17859 users.
![image](https://user-images.githubusercontent.com/20666568/59238055-05ca3400-8c2f-11e9-804b-f72e830b394a.png)

**Comment by johnnyjana730** [comment]
I also do the experiment on Movie and Restaurant datasets and the evaluation score of one with Label Smoothness Regularization still don't outperform steadily the one without Label Smoothness Regularization. I know the hyper-parameter settings are different compared to the paper. I just want to reproduce your experiment result and compare to mine. I just try my best to increase score of one with Label Smoothness Regularization as high as possible.
